xla/stream_executor/gpu/gpu_types.h

```diff
@@ -81,6 +82,8 @@ using GpuGraphNodeHandle = CUgraphNode;
 
 #endif
 
+using Dependencies = absl::InlinedVector<GpuGraphNodeHandle, 1>;
```

Comment:
I do actually prefer absl::Span in gpu_driver/types/cuda_driver because we have so many aliases, it makes it easier to see `absl::Span<GpuGraphNodeHandle>` and `absl::Span<CUgraphNodeHandle>` to make sure that types are correct.

Dependencies inside CommandBufferCmd can be aliased as it's not important what is their actual type is

Reply:
Fixed